content/docs/openshift/install-kubeflo.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "Install Kubeflow"
+description = "Instructions for deploying Kubeflow from the command line"
+weight = 4
++++
+
+This guide describes how to use the `kfctl` CLI to deploy Kubeflow 0.7 on an existing OpenShift 4.2 cluster.
+
+## Prerequisites
+
+### OpenShift 4 cluster
+
+* You need to have access to an OpenShift 4 cluster as `cluster-admin` to be able to deploy Kubeflow.
+* You can run a local cluster using [Code Ready Containers](https://code-ready.github.io/crc/), use https://try.openshift.com or bring your own cluster.
+* You will need to login as a user with `cluster-admin` privileges to the cluster to be able to deploy Kubeflow.
+* Install [`oc` command line tool](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html) to communicate with the cluster.
+
+#### Code Ready Containers
+
+If you are using Code Ready Containers, you will need to make sure you have enough resources configured for the VM:
+
+Recommended: 
+
+```
+16GB memory
+6 cpu
+5G disk space
+```
+
+Minimal:
+
+```
+10GB memory
+6 cpu
+30G disk space (default for CRC)
+```
+
+## Installing Kubeflow
+
+To install Kubeflow 0.7 on OpenShift 4.2 please follow the steps below:
+
+1. Clone the [opendatahub/manifests](https://github.com/opendatahub-io/manifests) fork repo which defaults to the branch `v0.7.0-branch-openshift`
+
+```
+git clone https://github.com/opendatahub-io/manifests.git
+cd manifests
+```
+
+2. Build the deployment configuration using the Openshift `KFDef` file and local downloaded manifests 
```

Comment:
Remove code formatting from KfDef. If you feel that this needs to be emphasized, consider bolding it. This is because KfDef refers to a type of file definition, rather than a  command.

Reply:
Yeah, I know about the list, but somehow I still prefer to keep the numbering in the MD:) But I changed it to keep consistent with your best practices, thanks